packages/labs/ssr/src/test/lib/render-lit_test.ts

```diff
@@ -218,6 +218,36 @@ test('element with property', async () => {
   );
 });
 
+test('element with attribute', async () => {
```

Comment:
Could you add a case for `'element with empty string attribute'` too?

Reply:
added 👍